dev/DemoApp1/plugin/wit/world.wit

```diff
@@ -3,11 +3,16 @@ package demoapp1:example;
 interface intf {
     use common:plugin/types.{error};
     helloworld2: func() -> result<string, error>;
+    multipli: func(a: u32, b: u32) -> string;
```

Comment:
I think you should return `result<string, error>`. I expect almost every function in every plugin to return `result<something, error>`.